### PR TITLE
feat: install the demo artifact in the data partition (`/data/www`)

### PR DESCRIPTION
**Pull request by tranchitella**
Ticket: MEN-6039
Changelog: Title

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Review by kjaskiewiczz — APPROVED** [review]
looks good

File: Dockerfile [hunk 1]
```diff
@@ -55,11 +55,11 @@ RUN directory-artifact-gen \
...
     -t raspberrypi4 \
     -t raspberrypi \
-    -d /var/www/localhost \
+    -d /data/www/localhost \
     -o mender-demo-artifact.mender \
-    /var/www/localhost \
+    /data/www/localhost \
     -- \
-    --software-filesystem rootfs \
+    --software-filesystem data-partition \
```
Comment (lluiscampos) [inline Dockerfile:62]: Should this be `data` instead?

**Review by lluiscampos — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (tranchitella) [reply Dockerfile:62]: Do we have an explicit policy/guideline about the name of the data partition filesystem?
I was unsure about it, too. I like data, but here we mention `data-partition`:
https://docs.mender.io/artifact-creation/software-versioning/advanced-versioning#advanced-versioning

Comment (kjaskiewiczz) [reply Dockerfile:62]: to me it doesn't matter if it's `data` or `data-partition` as long as we're consistent, which, in this case, means we should use `data-partition` here or change the docs

Comment (lluiscampos) [reply Dockerfile:62]: Yes. This was more like an open question to make us all aware (I did pre-approve). I think is okey to merge as-is.

**Pull request merged**